{
  "mitigation": "Mitigation for this issue is either not available or the currently available options do not meet the Red Hat Product Security criteria comprising ease of use and deployment, applicability to widespread installation base or stability.",
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Out of support scope",
      "package_name": "unbound",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "unbound",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "unbound",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Affected",
      "package_name": "unbound",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "rhcos",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.2",
      "fix_state": "Affected",
      "package_name": "rhosp-rhel8-tech-preview/openstack-unbound",
      "cpe": "cpe:/a:redhat:openstack:16.2"
    },
    {
      "product_name": "Red Hat OpenStack Platform 17.1",
      "fix_state": "Affected",
      "package_name": "rhosp-rhel9/openstack-unbound",
      "cpe": "cpe:/a:redhat:openstack:17.1"
    },
    {
      "product_name": "Red Hat OpenStack Platform 18.0",
      "fix_state": "Affected",
      "package_name": "openstack-unbound-container",
      "cpe": "cpe:/a:redhat:openstack:18.0"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2024-08-07T02:00:00Z",
  "bugzilla": {
    "description": "unbound: NULL Pointer Dereference in Unbound",
    "id": "2303456",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2303456"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "2.8",
    "cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:N/I:N/A:L",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-476",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2024-43167",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2024-43167\nhttps://nvd.nist.gov/vuln/detail/CVE-2024-43167\nhttps://github.com/NLnetLabs/unbound/issues/1072\nhttps://github.com/NLnetLabs/unbound/pull/1073/files"
  ]
}